startup_runners/startup_runner.c

```diff
@@ -13,19 +13,20 @@
 #include <errno.h>
 #include <dlfcn.h>
 #include <err.h>
+#include <stdint.h>
 #include <unistd.h>
 
-#define BENCH_FUNC_NAME "run_iter"
+#include "libkruntime.h"
 
-/* from libkruntime */
-double clock_gettime_monotonic();
+#define BENCH_FUNC_NAME "run_iter"
 
 int
 main(void)
 {
     double    start_time = -1;
 
-    start_time = clock_gettime_monotonic();
+    krun_measure(0);
```

Comment:
Well, I'm asking for your opinion.

Take a look at the implementation of krun_measure.

for index 0, wallclock is measured last, for 1, first. But when you measure 1 there is an integrity check which will likely fail if you have not already read index 0.